src/azul/service/dynamo_data_access.py

```diff
@@ -22,7 +22,7 @@ class DynamoDataAccessor:
     """
 
     def __init__(self, endpoint_url=None, region_name=None):
-        self.dynamo_client = aws.dynamo(endpoint_url, region_name)
+        self.dynamo_client = aws.dynamodb_resource(endpoint_url, region_name)
```

Comment:
Small Nit: There might be a naming issue here, when I see `{aws_service}_client` I think of the boto clients. IIRC the boto3 resources don't wrap all the functionality of the clients. So if someone called `self.dynamo_client.query` there should be an error because the object is actually a boto `resource`. 

On a side note, it looks like the `resource` classes keep a context of the client as well under `aws_resource.meta.client` https://github.com/boto/boto3/blob/cf62d26cb0f28357c935d541ed3f0dae196cd26d/boto3/resources/base.py#L95

Reply:
Your PR does not have to change this btw, I just wanted to highlight it.

Reply:
Yep I also noticed the inconsistency. I don't want to address it here since that class doesn't seem to be actually used anywhere and I suspect the better call would be to remove it altogether.